packages/connect/src/connector/index.ts

```diff
@@ -1,5 +1,5 @@
-import { getConnectorClient as slGetConnectorClient } from "./smoldot-light.js"
-import { getConnectorClient as eGetConnectorClient } from "./extension.js"
+import { createScClient as slcreateScClient } from "./smoldot-light.js"
+import { createScClient as ecreateScClient } from "./extension.js"
```

Comment:
This is very error-prone. I suggest to change it to:
```suggestion
import { createScClient as smoldotScClient } from "./smoldot-light.js"
import { createScClient as extensionScClient } from "./extension.js"
```